bcgov/HOUS-permit-portal
# CHES Email Services

## Description
- Initial Implementation of `CHESApiWrapper` to use the CHES service to send emails in Openshift environments (https://ches.api.gov.bc.ca/api/v1/docs)
- Creates a `CustomDeviseMailer` an overrides Devise to use this so we can get it to send through CHES
- Keeps local functionality for emails going through `letter_opener`

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

BPHH-507

## Steps to QA
- Merge to `develop`
- Set the proper ENV vars
- Check if this works in the live environments
